src/blog/the-unbreakable-slides-for-your-corporate-communication.md

```diff
@@ -14,53 +14,51 @@ canonical: "https://medium.com/@david.dalbusco/app-shortcuts-and-maskable-icons-
+With this new feature, we aim to provide you with a new kind of editor for presentations and to assist you in the implementation of your brand's CI/CD into your slides. In close collaboration with your designer and adhering to your guidelines, we will create a set of unbreakable templates to which you can apply your design, logo, colors, fonts, styles and any other specifications. These templates will then be used by all your employees and collaborators going forward, thereby streamlining your company's visual identity.
 
-But, there is more and here are some of the main features we have in mind for companies!
+That's not all! Below are some other features we are rolling out for enterprise customers.
 
 *****
 
 #### More Than Static Content
 
-Templates are made with the web and have no other limits than what it can do. It notably means that if your business do offer web solutions, you can integrated these within your slides.
+Templates are made with the web and have no other limits than the web's own. It means that if your business offers web solutions, you can integrate these within your slides.
 
 *****
 
-#### Collaborate And Shared Library
+#### Collaborate And Share A Library
 
-There is no team without communication. That’s why, multiple users can edit a presentation at the same time and are able to express and exchange comments, without having to use multiple communications channels.
+Teams are nothing without good communication. That’s why multiple users can edit the same presentation in real time and share ideas and comments.
 
-Likewise, with the help of a company’s media library, collaborators are able to share their assets’ resources.
+What's more, collaborators can now upload and use your company's media resources and assets in their own slide decks.
 
 *****
 
 #### Always Up-To-Date
 
-Over time your corporate style will evolve. We can make the propagation of these changes easier.
+Over time, your corporate style will evolve. We simplify this stylistic growth.
 
 *****
 
 #### Custom Domain, In Housing and Share Privately
 
-You can publish your decks under your own domain name. It is also possible to create private links to prevent access to your published content.
+You can publish your decks under your own domain name. It's also possible to create authenticated links for private content.
 
-If you rather like to integrate DeckDeckGo in your infrastructure and private network rather than using our SASS platform, we are open to the ideas too.
+Finally, if you prefer to integrate DeckDeckGo into your infrastructure and private network instead of using our SaaS platform, we are here to help.
 
 *****
 
 ### And So Much More
 
-These above main features are the one we think are specific to business but DeckDeckGo is so much more than that.
+That rounds up the new features for our enterprise customers – but we're also releasing a ton of new things for private users!
```

Comment:
That rounds up the new features for our enterprise customers – but we have already released a ton of things for any users!